src/main/java/pl/uj/io/cuteanimals/model/action/ThrowAwayAction.java

```diff
@@ -0,0 +1,58 @@
+package pl.uj.io.cuteanimals.model.action;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import pl.uj.io.cuteanimals.model.GameState;
+import pl.uj.io.cuteanimals.model.Result;
+import pl.uj.io.cuteanimals.model.interfaces.IAction;
+import pl.uj.io.cuteanimals.model.interfaces.ICharacter;
+import pl.uj.io.cuteanimals.model.interfaces.IItem;
+import pl.uj.io.cuteanimals.model.interfaces.IResult;
+
+public class ThrowAwayAction implements IAction {
+    private List<String> args;
+
+    public ThrowAwayAction() {
+        this.args = new ArrayList<>();
+    }
+
+    @Override
+    public IResult execute(ICharacter character) {
+        if (!getAcceptableStates().contains(character.getCurrentGameState())) {
+            return new Result("This action cannot be executed now");
```

Comment:
Maybe change to sth like this:
```suggestion
            return new Result("This isn't time for that.");
```
Just to keep immersion.